Trakt/Model/TraktAudio.cs

```diff
@@ -0,0 +1,82 @@
+using System.ComponentModel;
+
+namespace Trakt.Model
+{
+    /// <summary>
+    /// Enum TraktAudio.
+    /// </summary>
+    public enum TraktAudio
+    {
+        /// <summary>
+        /// LPCM audio.
+        /// </summary>
+        [Description("lpcm")]
```

Comment:
I don't believe the `Description` overrides what is returned in the `.ToString()` call

Reply:
You're correct I misread the stackoverflow. I'll switch that back and supress the warning.